sunpy/tests/pytest_intercept_remote/tests/test_plugin_intercept_remote.py

```diff
@@ -0,0 +1,111 @@
+import socket
+from urllib.request import urlopen
+
+import pytest
+import requests
+
+
+@pytest.fixture(scope="class")
+def skip_condition(request):
+    if request.config.option.intercept_remote:
+        pytest.skip("rerun without --intercept-remote option")
+
+
+@pytest.mark.usefixtures("skip_condition")
+class TestInterceptPlugin():
```

Comment:
Does this need to a class?

Reply:
It doesn't have to be but its easier to apply fixture over a class, what's your opinion?

Reply:
If there is no reason for it to be a class, I would prefer it to be removed.